### ISSUE TITLE
Highway network relu instead of sigmoid

### ISSUE THREAD

**Issue by haikalzain** [BlackStar1313/ICDAR-2019-RRC-SROIE#4]
I noticed the highway network uses a ReLU instead of sigmoid for transform gate

`        self.affine_transformation = torch.nn.Sequential(
            torch.nn.Linear(input_size, input_size, bias=True),
            torch.nn.ReLU()
        ).apply(HighwayNetwork.init_affine_transformation)

        self.transform_gate = torch.nn.Sequential(
            torch.nn.Linear(input_size, input_size, bias=True),
            torch.nn.ReLU()
        ).apply(HighwayNetwork.init_transform_gate)

    def forward(self, y):
        t = self.transform_gate(y)
        g = self.affine_transformation(y)
        # z = t * g(Why + bh) + (1 - t) * y
        z = t * g + (1.0 - t) * y
        return z`
